app/views/course/assessment/question/voice_responses/edit.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+question = @voice_response_question
+question_assessment = @question_assessment
+
+json.partial! 'form', locals: {
+  question: question,
```

Comment:
Do we need line 6 to 8?

Reply:
after I understand what it actually entails, I concluded it's not needed. I have removed it in the commit that will follow